includes/admin/reporting/reports.php

```diff
@@ -2548,377 +2547,454 @@ function display_export_report() {
+										)
+									);
 
 								wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' );
 								?>
-                                <input type="hidden" name="edd-export-class" value="EDD_Batch_Payments_Export"/>
-								<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
-                            </form>
-                        </div>
-                    </div>
-
-                    <div class="postbox edd-export-taxed-orders">
-                        <h3 class="hndle"><span><?php esc_html_e( 'Export Taxed Orders', 'easy-digital-downloads' ); ?></span></h3>
-                        <div class="inside">
-                            <p><?php esc_html_e( 'Download a CSV of all orders, taxed by Country and/or Region.', 'easy-digital-downloads' ); ?></p>
-                            <form id="edd-export-taxed-orders" class="edd-export-form edd-import-export-form" method="post">
-								<div class="edd-from-to-wrapper">
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Payments_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-taxed-orders">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Taxed Orders', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of all orders, taxed by Country and/or Region.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-taxed-orders" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-from-to-wrapper">
+								<label for="edd-taxed-orders-export-start" class="screen-reader-text"><?php esc_html_e( 'Export Taxed Orders Start Date', 'easy-digital-downloads' ); ?></label>
+									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-taxed-orders-export-start',
+											'class'       => 'edd-export-start',
+											'name'        => 'start',
+											'placeholder' => _x( 'From', 'date filter', 'easy-digital-downloads' )
+										)
+									);
+									?>
+								<label for="edd-taxed-orders-export-end" class="screen-reader-text"><?php esc_html_e( 'Export Taxed Orders End Date', 'easy-digital-downloads' ); ?></label>
 									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-taxed-orders-export-end',
+											'class'       => 'edd-export-end',
+											'name'        => 'end',
+											'placeholder' => _x( 'To', 'date filter', 'easy-digital-downloads' )
+										)
+									);
+									?>
+							</div>
+							<label for="edd_taxed_orders_export_status" class="screen-reader-text"><?php esc_html_e( 'All Statuses or Select Status', 'easy-digital-downloads' ); ?></label>
```

Comment:
I know this `select` field came with `-`, but I would change it to `_` for the same reasons above.